odoo_module_migrate/migration_scripts/migrate_100_110.py

```diff
@@ -46,7 +46,7 @@
             "report.external_layout_footer is obsolete.",
     },
     ".xml": {
-        "<tree(\n|.|\t)*color=\"[^>]*":
+        "<tree[ \n\t][^>]*color=":
```

Comment:
Shouldn't it be colors?

See #27 

Also, there could be other attributes between tree and color

Reply:
Yes, it should have been "colors". I have fixed that and added "fonts" since that's also deprecated.

[^>]* should allow anything except end of tag (>) between "<tree " and "colors=". So we would also warn about <tree mycolor="...">, but I guess there is no legal attributes ending with "colors" or "fonts", so I think that's acceptable.